lemur/common/utils.py

```diff
@@ -55,6 +55,18 @@ def get_psuedo_random_string():
     return challenge
 
 
+def get_random_secret(length):
```

Comment:
we should consider consolidating the `get_random_secrets()` and  `get_psuedo_random_string()`
for instance
```
def get_psuedo_random_string():    
"""    Create a random and strongish challenge.    """
    get_random_secret(24)
```

I have also noticed the `get_random_secret(length)` in other corners of the code:
https://github.com/Netflix/lemur/blob/176417c2400dfc60356769b26b4cee7421a91c91/docker/src/lemur.conf.py#L20-L24
https://github.com/Netflix/lemur/blob/d4e742ebf6ec66d2056d1d2da4b7b830570a6f6c/lemur/tests/conf.py#L11-L16

We can take care of it in a separate PR ;)